enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/AuthProceduresTest.java

```diff
@@ -957,17 +1038,17 @@ public static void testCallFail( GraphDatabaseAPI db, AuthSubject subject, Strin
         catch ( Exception e )
         {
             assertEquals( expectedExceptionClass, e.getClass() );
-            assertTrue( "Exception should contain '" + partOfErrorMsg + "'",
+            assertTrue( "Exception should contain '" + partOfErrorMsg + "', but was '" + e.getMessage() + "'",
```

Comment:
Maybe use `assertThat()` with `containsString()`, it can give nice message itself